charts/prometheus-mongodb-exporter/Chart.yaml

```
@@ -15,4 +15,4 @@ maintainers:
 name: prometheus-mongodb-exporter
 sources:
 - https://github.com/percona/mongodb_exporter
-version: 3.1.3
+version: 3.1.4
```

New feature should bump minor version.
```suggestion
version: 3.2.0
```